.github/workflows/firmware_build_test.yml

```diff
@@ -30,9 +30,9 @@ jobs:
         fetch-depth: 0
         submodules: recurvise
     - name: Download MAVSDK
-      run: wget https://github.com/mavlink/MAVSDK/releases/download/v0.33.1/mavsdk_0.33.1_ubuntu20.04_amd64.deb
+      run: wget https://github.com/mavlink/MAVSDK/releases/download/v0.36.0/mavsdk_0.36.0_ubuntu20.04_amd64.deb
     - name: Install MAVSDK
-      run: dpkg -i mavsdk_0.33.1_ubuntu20.04_amd64.deb
+      run: dpkg -i mavsdk_0.36.0_ubuntu20.04_amd64.deb
```

Comment:
Maybe jump in to 0.37 already?

Reply:
@TSC21 We are running the tests inside firmware, so I think this will not work unless firmware's test code is using (it is currently on 0.36)

Reply:
I think it's unrelated, so it will work. We can also bump on the firmware side respectively.

Reply:
Done!